src/Evolution/Systems/Cce/InterfaceManagers/GhLockstep.cpp

```diff
@@ -12,26 +12,26 @@
 #include "Parallel/CharmPupable.hpp"
 #include "Time/TimeStepId.hpp"
 
-namespace Cce {
+namespace Cce::InterfaceManagers {
 
-std::unique_ptr<GhWorldtubeInterfaceManager>
-GhLockstepInterfaceManager::get_clone() const noexcept {
-  return std::make_unique<GhLockstepInterfaceManager>(*this);
+std::unique_ptr<GhInterfaceManager> GhLockstep::get_clone() const noexcept {
+  return std::make_unique<GhLockstep>(*this);
 }
 
-void GhLockstepInterfaceManager::insert_gh_data(
+void GhLockstep::insert_gh_data(
     TimeStepId time_id, tnsr::aa<DataVector, 3> spacetime_metric,
     tnsr::iaa<DataVector, 3> phi, tnsr::aa<DataVector, 3> pi,
     const tnsr::aa<DataVector, 3> /*dt_spacetime_metric*/,
     const tnsr::iaa<DataVector, 3> /*dt_phi*/,
     const tnsr::aa<DataVector, 3> /*dt_pi*/) noexcept {
+  // NOLINTNEXTLINE(performance-move-const-arg)
```

Comment:
Probably better to move down one line

Reply:
Sorry, I don't understand -- the clang-tidy error is about the `time_id` that it characterizes as trivially copyable, which is in line 28.

Reply:
This just displayed strangely on mobile, showed a blank line, just ignore the comment :)